diff --git a/engelmann/CMakeLists.txt b/engelmann/CMakeLists.txt
index f92635a..0d9b73b 100644
--- a/engelmann/CMakeLists.txt
+++ b/engelmann/CMakeLists.txt
@@ -27,11 +27,11 @@ message("Glog include dir: ${GLOG_INCLUDE_DIRS}")
 message("Glog library: ${GLOG_LIBRARIES}")
 include_directories(${GLOG_INCLUDE_DIRS})
 
-find_package(SuiteSparse REQUIRED)
-message("SuiteSparse include dirs:")
-message("  cholmod: ${CHOLMOD_INCLUDE_DIR}")
-message("  csparse: ${CSPARSE_INCLUDE_DIR}")
-include_directories(${CHOLMOD_INCLUDE_DIR})
+#find_package(SuiteSparse REQUIRED)
+#message("SuiteSparse include dirs:")
+#message("  cholmod: ${CHOLMOD_INCLUDE_DIR}")
+#message("  csparse: ${CSPARSE_INCLUDE_DIR}")
+#include_directories(${CHOLMOD_INCLUDE_DIR})
 
 find_package(Ceres REQUIRED)
 message("Ceres include dir: ${Ceres_INCLUDE_DIRS}")
