diff --git a/CMakeLists.txt b/CMakeLists.txt
index ac07a15..8997031 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,7 @@ project(GCPR_shape_priors)
 
 add_compile_options(-std=c++0x -fPIC -msse3 -Wall -fopenmp -Wno-unused-variable)
 set(CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/cmake_modules/" ${CMAKE_MODULE_PATH})
-
+set(CUDA_USE_STATIC_CUDA_RUNTIME OFF)
 # Find Packages
 find_package(Ceres REQUIRED)
 include_directories(${CERES_INCLUDE_DIRS})
diff --git a/external/viz/CMakeLists.txt b/external/viz/CMakeLists.txt
index f177d7f..da1349e 100644
--- a/external/viz/CMakeLists.txt
+++ b/external/viz/CMakeLists.txt
@@ -26,7 +26,7 @@ set(VIZ_VERSION 0.0.1)
 
 set(CMAKE_INCLUDE_CURRENT_DIR ON)
 set(CMAKE_INCLUDE_CURRENT_DIR_IN_INTERFACE ON)
-
+set(CUDA_USE_STATIC_CUDA_RUNTIME OFF)
 # Compiler options
 add_compile_options(-std=c++0x -fPIC -msse3 -Wall -fopenmp -Wno-unused-variable)
 
diff --git a/external/viz/include/viz/viz.h b/external/viz/include/viz/viz.h
index 015e4ec..cf08aa5 100644
--- a/external/viz/include/viz/viz.h
+++ b/external/viz/include/viz/viz.h
@@ -12,8 +12,8 @@
 #include <vtkSmartPointer.h>
 
 // Eigen includes
-#include <eigen/Core>
-#include <eigen/Dense>
+#include <Eigen/Core>
+#include <Eigen/Dense>
 
 // Own includes
 #include "geometry.h"
diff --git a/external/viz/src/viz.h b/external/viz/src/viz.h
index 015e4ec..cf08aa5 100644
--- a/external/viz/src/viz.h
+++ b/external/viz/src/viz.h
@@ -12,8 +12,8 @@
 #include <vtkSmartPointer.h>
 
 // Eigen includes
-#include <eigen/Core>
-#include <eigen/Dense>
+#include <Eigen/Core>
+#include <Eigen/Dense>
 
 // Own includes
 #include "geometry.h"
